enterprise/views.py

```diff
@@ -202,18 +205,30 @@ def get_course_specific_consent(self, request, course_id):
         platform_name = configuration_helpers.get_value("PLATFORM_NAME", settings.PLATFORM_NAME)
         course_name = course_details['name']
         context_data = self.get_default_context(customer, platform_name)
+        # Translators: bold_start and bold_end are HTML tags for specifying
+        # enterprise name in bold text.
         course_specific_context = {
             'consent_request_prompt': _(
                 'To access this course and use your discount, you must first consent to share your '
-                'learning achievements with {enterprise_customer_name}.'
+                'learning achievements with {bold_start}{enterprise_customer_name}{bold_end}.'
             ).format(
-                enterprise_customer_name=customer.name
+                enterprise_customer_name=customer.name,
+                bold_start='<b>',
+                bold_end='</b>',
             ),
             'confirmation_alert_prompt': _(
-                'In order to start this course and use your discount, you must consent to share your '
-                'course data with {enterprise_customer_name}.'
+                'In order to start this course and use your discount, {bold_start}you must{bold_end} consent '
+                'to share your course data with {enterprise_customer_name}.'
             ).format(
-                enterprise_customer_name=customer.name
+                enterprise_customer_name=customer.name,
+                bold_start='<b>',
+                bold_end='</b>',
+            ),
+            'confirmation_alert_prompt_warning': _(
+                'If you do not consent to share your course data, that information may be shared with '
```

Comment:
This text isn't 100% clear to me; I understand what it means because I'm constantly immersed in the consent process, but a user might misunderstand it to mean that course data will be shared anyway. Unless you got this text directly from Marketing/edX, try the following:

> If you decline to share your course usage data, the fact that you declined to provide consent may be shared with {enterprise_customer_name}.